compiler/rustc_session/src/options.rs

```diff
@@ -1161,6 +1161,9 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "whether ELF relocations can be relaxed"),
     relro_level: Option<RelroLevel> = (None, parse_relro_level, [TRACKED],
         "choose which RELRO level to use"),
+    remap_rust_src_base: Option<PathBuf> = (None, parse_opt_pathbuf, [TRACKED_NO_CRATE_HASH],
```

Comment:
Can you rename this to `simulate_remapped_rust_src_base` and mention in the description that this option is meant to be used only for testing purposes. Also, I would make it `[TRACKED]` to be on the safe side.